packages/fxa-content-server/app/scripts/lib/sentry.js

```diff
@@ -76,31 +69,6 @@ function beforeSend(data) {
   return data;
 }
 
-/**
- * Determines if the error message should be sent
- * Source: github.com/getsentry/raven-js/blob/0184ca3bc7624be0fb0b093d9a96becc424bc9b5/src/raven.js
- *
- * @param {Object} data Error message data
- * @returns {Boolean} true if message should be sent.
- */
-function shouldSendCallback(data) {
-  if (data && data.message) {
-    var sameError = data.message === PREVIOUS_ERROR;
-    if (sameError) {
-      SAME_ERROR_ATTEMPTS++;
-    } else {
-      SAME_ERROR_ATTEMPTS = 0;
-    }
-    if (SAME_ERROR_LIMIT === SAME_ERROR_ATTEMPTS) {
```

Comment:
No need for this sort of limiting anymore?

Reply:
This was an option specific to raven, removed on line 184